cli/cmd/start/start.go

```diff
@@ -116,15 +126,24 @@ func StartCmd() *cobra.Command {
 			for _, merr := range res.Errors {
 				logger.Errorf("%s: %s", merr.FilePath, merr.Message)
 			}
-			for _, path := range res.AffectedPaths {
-				logger.Infof("Reconciled: %s", path)
+			for _, p := range res.AffectedPaths {
+				logger.Infof("Reconciled: %s", p)
 			}
 			logger.Infof("Hydration completed!")
 
+			installId, err := initGlobalConfig()
+			if err != nil {
+				return err
+			}
+			_, isDev := os.LookupEnv("RILL_IS_DEV")
 			// Create config object to serve on /local/config
 			localConfig := map[string]any{
 				"instance_id": localInstanceID,
 				"grpc_port":   grpcPort,
+				"install_id":  installId,
+				// TODO: do we need to only get the last folder?
+				"project_id": fmt.Sprintf("%x", md5.Sum([]byte(olapDSN))),
```

Comment:
I would make this `"project_path": filepath.Abs(repoDSN)`. Then the frontend also gets access to the full path if it ever wants to show it to the user.

The md5 hashing should happen on the frontend just before sending the analytics. I agree with the question that for analytics, you probably just want to hash the last folder name.